### ISSUE TITLE
Create "Talk" Pages for each course

### ISSUE THREAD

**Issue by AndrewMagliozzi** [FinalsClub/karmaworld#381]
We'll do this the easy way, using the Facbook API.

https://developers.facebook.com/docs/graph-api/reference/v2.2/app/groups

For each course, we'll create a public Facebook group.  On each course page, we'll provide a link to that group.

**Comment by btbonval** [comment]
Actually I think this might be a good application of Etherpad, because no revision history is necessary for a talk page.

The requirement of being a Facebook user might cause difficulties, but maybe not. You'll have a good sense of that from your campus reps. Just tossing out an alternative because I personally do not use Facebook.

**Comment by AndrewMagliozzi** [comment]
I think FB is a good enough solution. I actually got the idea from my call with out Brown reps.

**Comment by btbonval** [comment]
Sounds good then! The reps are closer to the audience than I'll ever be.

I'm an old man. The youth play their music is too loud and children need to stay off my lawn and so forth.

**Comment by btbonval** [comment]
Should Facebook groups be created when a course is created, or when the talk button on a course is pressed?

**Comment by AndrewMagliozzi** [comment]
I like the idea of a user doing something to create each of these groups.
Ideally we can create a user experience that makes intuitive sense.

If there is no group page, the prompt would say something like "create a
Facebook Group for {{ course name}}"...

If the group is created and the link is already in our database, then we'll
render a different prompt for users to "visit the Facebook Group for {{
course name }}"

Make sense?

**Comment by btbonval** [comment]
Yeah, that makes sense. So on the Course page, perhaps near "Add notes", we'd have this button which says either "Create a group on Facebook" or "Check out the Facebook group" depending on the situation.

Would we want a similar button on the notes pages alongside thank note, report note, download note, edit tags, edit this note? That way there'd be an easy-to-access button to get to the corresponding Facebook group from lecture notes.

**Comment by AndrewMagliozzi** [comment]
I think the group is at the course level entity. Let's leave individual notes aside. 

Though we may want a link to the class group on the header of each note page.

**Comment by btbonval** [comment]
Okay so Talk button only goes on the note-list page for the Course and there will be no Talk button on the note display page.

I think we had plans to add a Course link in some design patterns which never got fully approved or got lost in the design process or something. I had written a full breadcrumb design (All > School > Course > Note) before the pros came in. I wrote up a ticket for a link back to course in #445 , however it would be helpful if we had designs for a school landing page at some point for #257 .
